web_modules/Navigation/index.js

```diff
@@ -40,6 +40,14 @@ export default class Navigation extends Component {
               { "Plugins" }
             </a>
           </li>
+          <li className={ styles.item }>
+            <a
+              className={ styles.itemLink }
+              href="/documentation/api"
```

Comment:
This will trigger a full page reload. Please use Link component from statinamic
http://moox.io/statinamic/docs/faq/#navigation-links-with-active-class